code/model/SiteTree.php

```diff
@@ -1082,38 +1107,45 @@ public function canEdit($member = null) {
 	 * @return bool True if the current user can publish this page.
 	 */
 	public function canPublish($member = null) {
-		if(!$member || !(is_a($member, 'Member')) || is_numeric($member)) $member = Member::currentUser();
-		
-		if($member && Permission::checkMember($member, "ADMIN")) return true;
+		if(!$member || !(is_a($member, 'Member')) || is_numeric($member)) {
+			$member = Member::currentUser();
```

Comment:
https://github.com/silverstripe/silverstripe-cms/pull/1507/files

Check it in that view - weird.. Will download everything from now on :D